admin/class-acc_user_importer-admin.php

```diff
@@ -372,12 +716,41 @@ private function proccess_user_data ( $users ) {
 				$existingUser = get_user_by('email', $userEmail);
 				if( is_a( $existingUser, WP_User::class ) ) {
 					//We found a user. If the name is the same, update it.
-					//However if the name is different, it must be a family membership
-					//sharing the same email, so abort, let the current user be.
-					$this->log_dual(" > found by email, name is " . $existingUser->display_name);
+					//If the name is different, update the existing account if
+					//the incoming registration is a preferred one.
+					// In a family account, often the childs and partner have the
+					// same email address. We might have received the child record
+					// first and already created the account. If so, overwrite it
+					// with the parent (owner of email) information.
+					// If the name is different but membership type the same,
+					// do not overwrite the existing record.
+					// Note: another approach to avoid this complex code could be
+					// to reverse the processing order of the received user array.
+					// This way we would process older records first, and it
+					// seems we would receive the membership of adult1 first,
+					// then adult2, then childs. So naturally the owner of the
+					// account would be the first to be created.
 					if ($accUserData['display_name'] != $existingUser->display_name) {
```

Comment:
I just realized that we likely allow to update the display name, so this check falls apart 😅 Should not be your concern at this point though and it is already an issue with the existing plug-in.